dev/vtr_pretty_printers.py

```diff
@@ -0,0 +1,115 @@
+import re
```

Comment:
Add a module docstring and link to the using documentation?

Reply:
Done!